crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -21,6 +21,7 @@ pub struct CybersourcePaymentsRequest {
     processing_information: ProcessingInformation,
     payment_information: PaymentInformation,
     order_information: OrderInformationWithBill,
+    reference: String,
```

Comment:
I am not sure if this is correct. Feel free to request modification if needed. Thank you.

Reply:
@sahalsaad As per api docs of [cybersource](https://developer.cybersource.com/api-reference-assets/index.html#payments) , `code` (clientReferenceInformation->code ) is merchant reference. Can you please modify accordingly?

Reply:
Updated the code to `ClientReferenceInformation`. Feel free to let me know if still need another modification. This is the first time I writing rust code.